src/Phinx/Wrapper/TextWrapper.php

```diff
@@ -125,6 +125,26 @@ public function getMigrate($env = null, $target = null)
     }
 
     /**
+     * Returns the output from running the "seed:run" command.
+     * @param  string $env environment name (optional)
+     * @param  string $target target version (optional)
+     * @return string
+     */
+    public function getSeed($env = null, $target = null)
```

Comment:
If I can suggest, add a third parameter `$seed = null`, which would be the seed's name and is optional.